test/simple/test-module-version.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+require('../common');
+var assert = require('assert');
```

Comment:
We should be using `const` in master, definitely, but by itself that's not enough of a reason to update the test case if it's not using `const` already. Best to leave it unless there are other changes to make.